psoc/grobot_workspace/grobot_module_controller.cydsn/dht11.c

```diff
@@ -0,0 +1,234 @@
+//#include <project.h>
+
+#include "dht11.h"
+#include <stdint.h>
+#include <dht.h>
+
+#define null 0
+
+void delay(void);
+void unpack_dht_data(void);
+
+uint8_t check;
+uint8_t decode_flag;
+uint8_t decoded_temperature_data;
+uint8_t decoded_humidity_data;
+
+int temperature_values[8];
+int humidity_values[8];
+int diff_count_values[44]={0};
+int countvalues_datatimer[44]={0};
+int executed;
+int count;
+int risingedge_counts=0;
+
+//_CY_ISR_PROTO(Delay_Timer_ISR_Handler);
+//_CY_ISR_PROTO(DHT_Pin_ISR_Handler);
+
+//What Return_Decoded_Temperature does:
+//    -Returns the data in decoded_temperature_data
+// 
+//Parameters: None
+//Returns: uint8_t decoded_temperature_data
+
+uint8_t dht_get_temperature(){
+    
+    if(decoded_temperature_data == null){
```

Comment:
Use spaces between parentheses, e.g. ```if (!decoded_temperature_data) {```. 
(You don't have to do the comparison explicitly either.)